app/frontend/src/assets/images/images.tsx

```diff
@@ -160,6 +194,109 @@ const LogoWordmark = svgWrapper(LogoWordmarkP, {
   desc: "platform name and logo with a word play",
 });
 
+// Credits Page Icons
+const iconArrowDownSvg = svgWrapper(icongraphyArrowDown, {
+  title: "Arrow Down Icon",
+  desc: "",
+});
+const iconArrowLeftSvg = svgWrapper(icongraphyArrowLeft, {
+  title: "Arrow Left Icon",
+  desc: "",
+});
+const iconCarouselSvg = svgWrapper(icongraphyCarousel, {
+  title: "Carousel Icon",
+  desc: "",
+});
+const iconCheckSvg = svgWrapper(icongraphyCheck, {
+  title: "Check Icon",
```

Comment:
I saw there was already some wrapping in this file so I thought it was an CTJ project accessibility requirement and tried to do the same to keep it consistent. I noticed that the difference is when you hover over the svg, a tooltip would show the title explaining what the svg is about. However, this info is listed on each Card, so personally I don't see much value in wrapping these. I plan to delete these because I will reuse the existing icons.

Reply:
Ahh, I see. I mainly made the wrapper to reuse accessibility text that are not likely to change. Logos were a good candidate, since a logo will presumably still be a logo if used in a different page. Icons are bit iffier, since the meaning depends on its function.

Maybe a comment to clarify the purpose of the wrapper would be helpful?

Reply:
That makes total sense! The accessibility text can be helpful for logos because there is not always text next to them explaining what they are, whereas the credits page explains in detail what each icon is and where it's being used so I will remove the new wrapping I added and reuse existing images. I can also add a comment in the file explaining the purpose of the wrapper.